frontend/plan/components/DropdownButton.tsx

```diff
@@ -1,14 +1,27 @@
 import React, { useState } from "react";
-import PropTypes from "prop-types";
-import { useOnClickOutside } from "pcx-shared-components/src/useOnClickOutside";
+import { useOnClickOutside } from "../../shared-components/src/useOnClickOutside";
+
+interface DropdownButtonProps {
+    title: string;
+    children: Node;
+    filterData:
```

Comment:
Since this type annotation is copy/pasted, would be good to pull it out into a named type, like `FilterType` or something.